services/blockchain-indexer/jobs/dataService/validators.js

```diff
@@ -47,4 +48,19 @@ module.exports = [
 			}
 		},
 	},
+	{
+		name: 'validate.validators',
+		description: 'Validate cached validators and reload cache if necessary',
+		schedule: '0 * * * *', // Every hour
```

Comment:
Can we add `interval`/`schedule` support in the config as in https://github.com/LiskHQ/lisk-service/blob/release/0.7.0/services/transaction-statistics/jobs/transactionStatistics.js#L31-L32.
```suggestion
		description: 'Validate cached validators and reload cache if necessary.',
		schedule: '4-/15 * * * *', // Every 15 minutes starting at 4 minutes past the hour
```